docs/DeveloperGuide.md

```diff
@@ -1,38 +1,120 @@
 # Developer Guide
+## Product Name
+**WellNUS++**
 
 ## Acknowledgements
-
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+1. Reference to AB-3 Developer Guide: https://se-education.org/addressbook-level3/DeveloperGuide.html
+2. Reference to AB-3 diagrams code: https://github.com/se-edu/addressbook-level3/tree/master/docs/diagrams
+
+## Setting up, getting started
+### Setting up the project in your computer
+Firstly, **fork** this repo, and **clone** the fork into your computer. <br>
+<br>
+If you plan to use Intellij IDEA (highly recommended): <br>
+1. **Configure the JDK**: Follow the guide 
+[[se-edu/guides] IDEA: Configuring the JDK](https://se-education.org/guides/tutorials/intellijJdk.html) 
+to ensure Intellij is configured to use **JDK 11**.
+2. **Import the project as a Gradle project**: Follow the guide 
+[[se-edu/guides] IDEA: Importing a Gradle project](https://se-education.org/guides/tutorials/intellijImportGradleProject.html) 
+to import the project into IDEA.<br>
+   **Note**: Importing a Gradle project is slightly different from importing a normal Java project.
+3. **Verify the setup:**
+   1. Run the ```wellnus.WellNus``` and try a few commands.
+   2. [Run the tests](https://se-education.org/addressbook-level3/Testing.html) to ensure they all pass.
+
+### Before writing code
+1. **Configure the coding style**<br>
+If using IDEA, follow the guide 
+[[se-edu/guides] IDEA: Configuring the code style](https://se-education.org/guides/tutorials/intellijCodeStyle.html) 
+to set up IDEA’s coding style to match ours.<br>
+<br>
+2. **Set up CI**<br>
+   This project comes with a GitHub Actions config files (in `.github/workflows` folder). 
+When GitHub detects those files, it will run the CI for your project automatically at each push 
+to the `master` branch or to any PR. No set up required.<br>
+<br>
+3. **Learn the design**<br>
+When you are ready to start coding, we recommend that you get some sense of the overall design by reading about 
+[AddressBook’s architecture](https://se-education.org/addressbook-level3/DeveloperGuide.html#architecture).<br>
+<br>
```

Comment:
This may not be applicable to us. Contextualize it to WellNUS++. We should do a rough architecture flow for WellNUS++ in the near future

Reply:
I converted this to the following: 
![image](https://user-images.githubusercontent.com/89572803/226386535-21c1c45a-f47d-494a-94a5-d8e9c024b602.png)